src/modules/translation/languages/es.json

```
@@ -1392,7 +1392,11 @@
     "add_item": "Agregar Item",
     "limit_reached": "Solo puede crear {max} items por transaction",
     "no_items": "No tienes items disponibles para crear en la colleción \"{name}\"",
-    "unsynced_warning": "Algunos items tienen cambios que aún no se han aprobado."
+    "unsynced_warning": "Algunos items tienen cambios que aún no se han aprobado.",
+    "next": "Siguiente",
+    "total_items": "Total de items para mintear",
+    "confirm_title": "Estas por mintear  {count} {count, plural, un {item} other {items}}, cada item va a ser minteado a la dirección indicada con la cantidad seleccionada.",
```

The `{count, plural, one` is part of the logic to pluralize text:
```suggestion
    "confirm_title": "Estas por mintear {count} {count, plural, one {item} other {items}}, cada item va a ser minteado a la dirección indicada con la cantidad seleccionada.",
```